.gitignore

```diff
@@ -0,0 +1 @@
+tmp/
```

Comment:
Another drive-by comment: the red circle means this file is missing a final newline.  Can you fix that in the next revision?

(We have code checks that make sure we're not committing source files without final newlines, but they don't check 100% of the things in the source directory, so this new file you're adding doesn't seem to be checked.)